extensions/schema-compare/src/schemaCompareResult.ts

```diff
@@ -362,4 +396,111 @@ export class SchemaCompareResult {
 		let service = azdata.dataprotocol.getProvider<azdata.SchemaCompareServicesProvider>(providerName, azdata.DataProviderType.SchemaCompareServicesProvider);
 		return service;
 	}
+
+	private GetDefaultDeploymentOptions() {
```

Comment:
What do you think about making a remote call to get the defaults? This way, if the default changes in DacFx, we don't have to make an update here.

Reply:
Hmm.. sounds like a good idea.

Reply:
Oh just realized one thing while making the change :) - even if we do that we will have to add for description and so on to show it - might not be good idea to pass on loc strings and so on. Let's discuss quickly tomorrow before I make the change.

Reply:
Done